cloud/aws-functions/orphaned_openshift_eip.py

```diff
@@ -0,0 +1,73 @@
+# Remove elastic ip.
+import logging
+import datetime
+import boto3
+from utils import get_regions_list
+
+
+def is_ip_to_release(client, ip):
+    try:
+        tags = ip["Tags"]
+    except KeyError as e:
+        print(f"There are no tags in the resource {ip['PublicIp']}")
+        return False
+
+    tags_dict = {item["Key"]: item["Value"] for item in tags}
+
+    if "team" not in tags_dict.keys() or (
+        "team" in tags_dict.keys() and tags_dict["team"] != "cloud"
+    ):
+        return False
+
+    if "delete-cluster-after-hours" not in tags_dict.keys():
+        return True
+    instance_lifetime = float(tags_dict["delete-cluster-after-hours"])
+    current_time = datetime.datetime.now().timestamp()
```

Comment:
Nit: Perhaps move these two lines to line 30 to be after the try catch and closer to the if , where they are used

Reply:
I can't see any problem to keep it like this. I mean it's just a couple of rows not hundreds :)